shared-helpers/__tests__/occupancyFormatting.test.tsx

```diff
@@ -1,122 +1,154 @@
 import React from "react"
 import { cleanup } from "@testing-library/react"
-import { occupancyTable } from "../src/occupancyFormatting"
-import { Listing, UnitType, UnitGroup } from "@bloom-housing/backend-core/types"
-
-const unitTypeSRO = { name: "SRO", numBedrooms: 0 } as UnitType
-const unitTypeStudio = { name: "studio", numBedrooms: 0 } as UnitType
-const unitTypeOneBdrm = { name: "oneBdrm", numBedrooms: 1 } as UnitType
-const unitTypeTwoBdrm = { name: "twoBdrm", numBedrooms: 2 } as UnitType
-const unitTypeThreeBdrm = { name: "threeBdrm", numBedrooms: 3 } as UnitType
-const unitTypeFourBdrm = { name: "fourBdrm", numBedrooms: 4 } as UnitType
-
-const unitGroups: Omit<UnitGroup, "id" | "listing" | "openWaitlist" | "amiLevels" | "listingId">[] = [
-  {
-    unitType: [unitTypeStudio, unitTypeOneBdrm],
-    minOccupancy: 1,
-    maxOccupancy: 2,
-  },
-  {
-    unitType: [unitTypeOneBdrm],
-    minOccupancy: 1,
-    maxOccupancy: 3,
-  },
-  {
-    unitType: [unitTypeTwoBdrm],
-    minOccupancy: 2,
-    maxOccupancy: 6,
-  },
-  {
-    unitType: [unitTypeTwoBdrm],
-    minOccupancy: 2,
-    maxOccupancy: undefined,
-  },
-  {
-    unitType: [unitTypeTwoBdrm],
-    minOccupancy: undefined,
-    maxOccupancy: 2,
-  },
-  {
-    unitType: [unitTypeFourBdrm],
-    minOccupancy: 1,
-    maxOccupancy: undefined,
-  },
-  {
-    unitType: [unitTypeSRO],
-    minOccupancy: undefined,
-    maxOccupancy: 1,
-  },
-  {
-    unitType: [unitTypeTwoBdrm],
-    minOccupancy: 1,
-    maxOccupancy: 1,
-  },
-  {
-    unitType: [unitTypeThreeBdrm],
-    minOccupancy: 3,
-    maxOccupancy: 3,
-  },
-  {
-    unitType: [unitTypeFourBdrm],
-    minOccupancy: undefined,
-    maxOccupancy: undefined,
-  },
-  {
-    unitType: [unitTypeTwoBdrm, unitTypeOneBdrm],
-    minOccupancy: 1,
-    maxOccupancy: 7,
-  },
-]
+import { occupancyTable, getOccupancyDescription } from "../src/occupancyFormatting"
```

Comment:
The backend is really the easiest side to resolve and this along with partners and UI-Components is difficult largely because of the differences with unit groups, so you have to favor current changes in a lot of cases like this file.